doc/source/whatsnew/v1.0.2.rst

```diff
@@ -64,7 +65,6 @@ Bug fixes
 
 **Datetimelike**
 
-- Bug in :meth:`DataFrame.reindex` and :meth:`Series.reindex` when reindexing with a tz-aware index (:issue:`26683`)
```

Comment:
moved to regressions, as this also fixed a regression (initially only a bug, but was backported later)